components/commercial_filter/views/popular_artists/index.styl

```diff
@@ -22,7 +22,7 @@
       white-space nowrap
       overflow hidden
       text-overflow ellipsis
-      max-width 175px
+      width calc(66% \- 15px)
```

Comment:
Good question, this is just subtracting 15px!

I see the other use cases of it seem to escape the subtraction symbol, and it does generate the correct CSCS.

Maybe the slash isn't necessary?